static/js/vue-cdr-access/src/components/full_record/fileList.vue

```diff
@@ -129,19 +134,21 @@ export default {
                 { type: 'file-size', targets: 3 },
                 {
                     render: (data, type, row) => {
+                        this.brief_object = row;
```

Comment:
I'm not sure I understand this change. It seems like we are storing the current row while iterating through the list of file records to a component wide variable. Is this just so the row doesn't have to be passed as a parameter into various helper methods, or is there another purpose? If not, I would prefer to stick with passing it around, since I worry there might be unexpected side effects of relying on an instance variable instead of using the loop variable, and it seems like a mismatch of scope

Reply:
It was so it didn't have to be passed around.